src/helperFunctions/yara_binary_search.py

```diff
@@ -12,23 +13,30 @@
 
 
 class YaraBinarySearchScanner:
+    '''
+    A scanner for searching with YARA rules on the binaries belonging to the firmware and file objects contained
```

Comment:
It think we can improve the wording here. How 'bout:

This class provides functionality to scan files in the database for [yara](http://virustotal.github.io/yara/) patterns. The public method allows to either match a given set of patterns on all files in the database or focus only on files included in a single firmware.

Reply:
I agree → changed